src/python/grpcio/grpc/_cython/_cygrpc/call.pyx.pxi

```diff
@@ -106,6 +107,8 @@ cdef class Call:
   def __dealloc__(self):
     if self.c_call != NULL:
       grpc_call_destroy(self.c_call)
+    grpc_shutdown()
```

Comment:
I was under the impression that every object's initialization matches its deinitialization ala C++-style RAII (but here across program rather than block scope). Short of a program freeze how would `g_initializations` never reach `0`?